File: applications/python/chat_completion/stream.py
from client import client
from config import MODEL

def stream():
    streamed_response = client.chat.completions.create(
        messages=[
            {
                "role": "user",
                "content": "Please repeat this and only this once: Hello Stream Test",
            }
        ],
        model=MODEL,
        stream=True
    )

    for chunk in streamed_response:
        if chunk.choices[0].delta.content is not None:
            yield chunk.choices[0].delta.content

File: applications/node/src/chat-completions/stream.ts
import client, { MODEL } from "@/client";

export async function stream(write: (str: string) => void): Promise<void> {
  const stream = await client.chat.completions.create({
    model: MODEL,
    messages: [
      {
        role: "user",
        content: "Please repeat this and only this once: Hello Stream Test",
      },
    ],
    stream: true,
  });

  for await (const chunk of stream) {
    write(chunk.choices[0]?.delta?.content ?? "");
  }
}
